src/helpers/serverErrors.ts

```diff
@@ -11,28 +11,28 @@ const serverError = (
 
   if (err.name === 'ValidationError') {
     return res.status(400).json({
-      data: { errors }
+      data: { errors },
     })
   }
 
   if (err.name === 'JsonWebTokenError') {
     return res.status(401).json({
-      message: 'Unauthorized'
+      message: 'Unauthorized',
     })
   }
 
   if (err instanceof CustomError) {
     if (status !== undefined) {
       return res.status(status).json({
-        message
+        message,
       })
     }
   }
-
+  console.log(err)
```

Comment:
do you need this or it was by mistake?

Reply:
I need it for debugging for now 
but I will remove it after we finish the backend.